### PR TITLE
fix(service-account): apply service account admin mode

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/asset-inventory/pages/ServiceAccountPage.vue [hunk 1]
```diff
@@ -100,11 +111,23 @@ const tableState = reactive({
...
+            return [{ name: ACCOUNT_TYPE.TRUSTED, label: ACCOUNT_TYPE_BADGE_OPTION[ACCOUNT_TYPE.TRUSTED].label }];
+        }
+        return [
+            { name: ACCOUNT_TYPE.GENERAL, label: ACCOUNT_TYPE_BADGE_OPTION[ACCOUNT_TYPE.GENERAL].label },
+            { name: ACCOUNT_TYPE.TRUSTED, label: ACCOUNT_TYPE_BADGE_OPTION[ACCOUNT_TYPE.TRUSTED].label },
+        ];
+    }),
     selectedAccountType: computed<AccountType>(() => serviceAccountSchemaState.selectedAccountType),
+    tableTitle: computed(() => {
+        console.debug('tableState.selectedProvider', state.selectedProvider);
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/pages/ServiceAccountPage.vue:125]: nitpick: console.debug

File: apps/web/src/services/asset-inventory/pages/ServiceAccountPage.vue [hunk 2]
```diff
@@ -100,11 +111,23 @@ const tableState = reactive({
...
+        if (state.selectedProvider === 'google_cloud') return PROVIDER_ACCOUNT_NAME.google_cloud;
+        if (state.selectedProvider === 'azure') return PROVIDER_ACCOUNT_NAME.azure;
+        if (state.selectedProvider === 'kubernetes') return PROVIDER_ACCOUNT_NAME.kubernetes;
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/pages/ServiceAccountPage.vue:128]: suggestion: how about 
```typescript
if (Object.keys(PROVIDER_ACCOUNT_NAME).include(state.selectedProvider) return PROVIDER_ACCOUNT_NAME[state.selectedProvider];
```

**Review by yuda110 — APPROVED** [review]
LGTM~

**Review by sulmoJ — APPROVED** [review]
Thank you so much!!

File: apps/web/src/services/asset-inventory/pages/ServiceAccountPage.vue (hunk 1 above)
Comment (piggggggggy) [reply apps/web/src/services/asset-inventory/pages/ServiceAccountPage.vue:125]: 😨

File: apps/web/src/services/asset-inventory/pages/ServiceAccountPage.vue (hunk 2 above)
Comment (piggggggggy) [reply apps/web/src/services/asset-inventory/pages/ServiceAccountPage.vue:128]: 😯

**Pull request merged**